Makefile

```diff
@@ -105,16 +118,6 @@ build-docs:
 # conservatively.
 
 
-# Run by CI, these commands should not modify files, but only check compliance.
-.PHONY: lint-ci
-lint-ci:
-	flake8
-	isort --check .
-	black --check --diff .
-	mypy conbench
-	mypy benchalerts
```

Comment:
Moving these two Makefile targets closer again. Slightly violates the rule of where to put which targets, but having those two close-by simply is more important.